src/components/HeaderMenu.tsx

```diff
@@ -57,6 +61,34 @@ const ExportMenuItem = () => {
 
 const HeaderMenu: FunctionComponent<{ onOpen: any }> = ({ onOpen }) => {
   const dispatch = useDispatch()
+  const toast = useToast()
+  const fontsToBeLoaded: Array<string> = []
+
+  const onActive = () => {
+    const uniqueFonts = uniq(fontsToBeLoaded)
+    uniqueFonts.forEach((font: string) =>
+      dispatch.app.addFonts(titleCase(font)),
+    )
+  }
+
+  const onInActive = () => {
+    toast({
+      title: 'Error while loading fonts',
+      status: 'error',
+      duration: 1000,
+      isClosable: true,
+      position: 'top',
+    })
+  }
+
+  const loadFontsOnImport = (theme: any) => {
+    if (theme.fonts)
+      Object.values(theme.fonts).forEach((font: any) =>
+        fontsToBeLoaded.push(font.toString()),
```

Comment:
@ashwinKumar0505 it might be a better idea to have a seperate key in workspace.json 
`"google fonts": ['nunito', 'open sans']`
this way you can easily load only these fonts on import (rather than doing unique, and checking if the font exists on browser etc